osu.Game/OsuGame.cs

```diff
@@ -182,7 +182,26 @@ private void load(FrameworkConfigManager frameworkConfig)
             // bind config int to database SkinInfo
             configSkin = LocalConfig.GetBindable<int>(OsuSetting.Skin);
             SkinManager.CurrentSkinInfo.ValueChanged += skin => configSkin.Value = skin.NewValue.ID;
-            configSkin.ValueChanged += skinId => SkinManager.CurrentSkinInfo.Value = SkinManager.Query(s => s.ID == skinId.NewValue) ?? SkinInfo.Default;
+            configSkin.ValueChanged += skinId =>
+            {
+                var skinInfo = SkinManager.Query(s => s.ID == skinId.NewValue);
+
+                if (skinInfo == null)
+                {
+                    switch (skinId.NewValue)
+                    {
+                        case -1:
+                            skinInfo = DefaultLegacySkin.Info;
```

Comment:
I don't like how skin info of legacy default is inside its class while lazer default is in `SkinInfo`, you should either move the default legacy info to `SkinInfo` or move default lazer info to its class (`DefaultSkin`) which seems better imo.

Reply:
why?

Reply:
actually nvm on this one, I was thinking differently about the default legacy skin